build.sbt

```diff
@@ -54,25 +63,24 @@ lazy val shapelessScanner = (project in file("shapeless"))
   .settings(
     name := s"shapeless-scanner",
     description := "Shapeless Scanner",
-    libraryDependencies += "com.chuusai" %% "shapeless" % "2.3.1"
+    libraryDependencies += DependsOn.shapeless
   )
   .dependsOn(core)
 
-lazy val benchmarks = (project in file("benchmarks"))
-  .settings(commonSettings: _*)
-  .settings(noPublishSettings: _*)
-  .settings(
-    name := s"$repo-benchmarks",
-    libraryDependencies += "com.storm-enroute" %% "scalameter-core" % "0.7" % Test
-  )
+lazy val benchmarks = Project(id = "benchmarks", base = file("benchmarks"),
+  settings = commonSettings ++
+             Seq(libraryDependencies += DependsOn.scalaMeter(scalaVersion.value)) ++
+             Seq(name := s"$repo-benchmarks") ++ noPublishSettings ++
+             Seq((skip in compile) := scalaVersion.value startsWith "2.12")
+)
   .dependsOn(core)
 
-lazy val root = (project in file("."))
-  .settings(commonSettings: _*)
-  .settings(docSettings: _*)
-  .settings(noPublishSettings: _*)
-  .settings(releaseSettings: _*)
-  .aggregate(core, akka, shapelessScanner)
+lazy val root =
+  Project(
+    id = "better-files",
+    base = file( "." ),
+    settings = Seq(name:= "Better Files") ++ commonSettings ++ docSettings ++ noPublishSettings ++ releaseSettings
```

Comment:
don't use the settings Project apply parameter, use the .settings method on Project instead.